src/handles/tpm.rs

```diff
@@ -37,6 +37,19 @@ pub enum TpmHandle {
     AttachedComponent(attached_component::AttachedComponentTpmHandle),
 }
 
+impl TpmHandle {
+    /// Method that indicates if the flushing the
+    /// context of the handle is a valid action.
+    pub fn may_be_flushed(&self) -> bool {
```

Comment:
Maybe we want this to be `pub(crate)` instead of fully `pub`? I don't think we want this as the public API per se?